RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -725,16 +725,16 @@ void AliAnalysisTaskAO2Dconverter::InitTF(ULong64_t tfId)
   // Associuate branches for V0s
   TTree* tV0s = CreateTree(kV0s);
   if (fTreeStatus[kV0s]) {
-    tV0s->Branch("fPosTrackID", &v0s.fPosTrackID, "fPosTrackID/I");
-    tV0s->Branch("fNegTrackID", &v0s.fNegTrackID, "fNegTrackID/I");
+    tV0s->Branch("fIndexTracks_Pos", &v0s.fIndexTracksPos, "fIndexTracks_Pos/I");
+    tV0s->Branch("fIndexTracks_Neg", &v0s.fIndexTracksNeg, "fIndexTracks_Neg/I");
```

Comment:
Do we really need the underscores?

Reply:
Yes, this is to identify in the merging script the end of the table name. We have mixed case, e.g. fIndexMcParticles_CustomSuffix